client/sttp-client/src/main/scala/tapir/client/sttp/EndpointToSttpClient.scala

```diff
@@ -11,8 +11,18 @@ import tapir.internal._
 
 class EndpointToSttpClient(clientOptions: SttpClientOptions) {
   def toSttpRequest[I, E, O, S](e: Endpoint[I, E, O, S], baseUri: Uri): I => Request[Either[E, O], S] = { params =>
+    toSttpRequestRaw(e, baseUri)(params).mapResponse(getOrThrow)
+  }
+
+  def toSttpRequestRaw[S, O, E, I](e: Endpoint[I, E, O, S], baseUri: Uri): I => Request[DecodeResult[Either[E, O]], S] = { params =>
```

Comment:
I think we need a method in the package object as well? + some comment explaining the difference (or rather, specifying that the `toSttpRequest` method can throw an exception).

I'm also thinking about naming ... Not sure about the `raw` :) Maybe `safe`?

Reply:
How about now?